src/main/java/fi/metatavu/dcfb/server/items/ItemController.java

```diff
@@ -234,14 +234,16 @@ public void deleteItemUsers(Item item) {
   /**
    * Searches items
    * 
+   * @param nearLat prefer items near geo point
+   * @param nearLon prefer items near geo point
    * @param categories filter by categories. Ignored if null
    * @param locations filter by locations. Ignored if null
    * @param search Search by free-text. Ignored if null
    * @param firstResult result offset
    * @param maxResults maximum number of results returned
    * @return search result
    */
-  public SearchResult<Item> searchItems(List<Category> categories, List<Location> locations, String search, UUID currentUserId, Long firstResult, Long maxResults, List<ItemListSort> sorts) {
+  public SearchResult<Item> searchItems(Double nearLat, Double nearLon, List<Category> categories, List<Location> locations, String search, UUID currentUserId, Long firstResult, Long maxResults, List<ItemListSort> sorts) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Method has 9 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=21&issues=AWVnq7tTXKENQn1Shuhm&open=AWVnq7tTXKENQn1Shuhm)

[ISSUE_KEY]: AWVnq7tTXKENQn1Shuhm